File: android/app/src/main/java/co/kiqit/kiqit/events/Event.java
package co.kiqit.kiqit.events;

/**
 * Created by safaiyeh on 8/17/16.
 */
public class Event {

    private String title;
    private String location;
    private String startTime;
    private String endTime;

    public Event(String title, String location, String startTime, String endTime) {
        this.title = title;
        this.location = location;
        this.startTime = startTime;
        this.endTime = endTime;
    }

    public String getTitle() {
        return title;
    }

    public String getLocation() {
        return location;
    }

    public String getStartTime() {
        return startTime;
    }

    public String getEndTime() {
        return endTime;
    }
}

File: ios/kiqit/Event.swift
//
//  Event.swift
//  kiqit
//
//  Created by Jason Safaiyeh on 7/26/16.
//  Copyright © 2016 Jason Safaiyeh. All rights reserved.
//

import Foundation

class Event {
    let title: String
    let description: String
    let location: String
    let startTime: String
    let endTime: String
    
    init(title: String, description: String, location: String, startTime: String, endTime: String) {
        self.title = title
        self.description = description
        self.location = location
        self.startTime = startTime
        self.endTime = endTime
    }
}
